### PR TITLE
Newsletter-220: Translate to French

### PR DESCRIPTION
**Pull request by Copinmalin**
Hello @jonatack and @bitschmidty,

You will find the translation of this newsletter 220,

Thanks for review and help,

### CODE REVIEW

File: _posts/fr/newsletters/2022-10-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+    ZmnSCPxj [a critiqué][zmnscpxj valve] plusieurs aspects de l'idée, y compris
+    le fait que ceux qui effectuent la dépense pourraient toujours envoyer la même quantité de
+    valeur par le biais d'un canal à taux maximum inférieur, ce qui aurait pour
+    conséquence de le déséquilibrer à nouveau, simplement en divisant un paiement
+    global en petites parties supplémentaires. Towns a suggéré que ce problème
+    pourrait être résolu en limitant le taux.
+
+    La discussion semblait être en cours au moment où ce résumé a été écrit, mais
+    nous nous attendons à ce que plusieurs nouveaux aperçus viennent dans les
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2022-10-05-newsletter.md:108]: Is this nous nous duplicate intentional?

Comment (jonatack) [reply _posts/fr/newsletters/2022-10-05-newsletter.md:108]: Yes, it's good (a reflexive verbe).

Comment (bitschmidty) [reply _posts/fr/newsletters/2022-10-05-newsletter.md]: Added a commit to bypass the duplicate word linter check

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin 

@jonatack if you have translation feedbacks we can tweak in a follow up PR

**Pull request merged**